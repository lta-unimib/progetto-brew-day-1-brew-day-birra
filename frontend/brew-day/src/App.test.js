import React from "react";
import ReactTestRenderer from 'react-test-renderer';
import App from './App';

describe('App component', () => {
  test('should render correctly', () => {
    const renderer = ReactTestRenderer.create(<App />);
    const json = renderer.toJSON();
    expect(json).toMatchSnapshot();
  });
});