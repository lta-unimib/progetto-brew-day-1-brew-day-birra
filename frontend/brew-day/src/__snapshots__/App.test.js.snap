// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component should render correctly 1`] = `
Array [
  <div>
    <ul
      id="navBar"
    >
      <li>
        <a
          href="/"
          onClick={[Function]}
        >
          <img
            id="logo-image"
            src="../icons/logo2.png"
          />
          <span
            id="logo-text"
          >
            Brew Day!
          </span>
        </a>
      </li>
      <li>
        <a
          href="/ricette"
          onClick={[Function]}
        >
          Ricette
        </a>
      </li>
      <li>
        <a
          href="/birre"
          onClick={[Function]}
        >
          Birre
        </a>
      </li>
      <li>
        <a
          href="/inventario"
          onClick={[Function]}
        >
          Inventario
        </a>
      </li>
      <li>
        <a
          href="/spesa"
          onClick={[Function]}
        >
          Spesa
        </a>
      </li>
    </ul>
  </div>,
  <div />,
]
`;
